bfd/src/test/java/gov/cms/ab2d/bfd/client/BFDClientConfigurationTest.java

```diff
@@ -1,38 +0,0 @@
-package gov.cms.ab2d.bfd.client;
```

Comment:
I guess the deletion was unintentional?

Reply:
I have no memory of this

Reply:
must have been accidental. Satheesh restored the file.